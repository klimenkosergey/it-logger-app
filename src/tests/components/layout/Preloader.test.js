import React from 'react';
import { shallow } from 'enzyme';

import Preloader from '../../../components/layout/Preloader';

it('should render Preloader correctly', () => {
  const wrapper = shallow(<Preloader />);
  expect(wrapper).toMatchSnapshot();
});
