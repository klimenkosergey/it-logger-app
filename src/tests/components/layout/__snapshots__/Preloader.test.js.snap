// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Preloader correctly 1`] = `
<div
  className="progress blue lighten-4"
>
  <div
    className="indeterminate blue"
  />
</div>
`;
